src/components/forms/form-elements/FormElements.vue

```diff
@@ -136,35 +136,56 @@
         <va-card :title="$t('forms.selects.title')">
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/61463229-a7564b00-a97c-11e9-9d59-40cdd640a672.png)

Shouldn't show clear icon if value is already cleared.